CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## edge-19.4.3
+
+* CLI
+  * **Fixed** `linkerd upgrade` command not upgrading proxy containers (thanks
+    @jon-walton for the issue report!)
+  * **Fixed** `linkerd upgrade` command not installing the identity service when
+    it was not already installed
+* Internal
+  * Updated web component to read the UUID from the `linkerd-config` ConfigMap
+    (thanks @KatherineMelnyk!)
+  * Replaced UUID generation with a more secure version
```

Comment:
might phrase this as "Eliminate false-positive vulnerability warnings related to go.uuid"

The point being that our use of uuids has no security implications one way or the other.